lambdas/process-journey-step/src/main/resources/statemachine/build-statemachine-config.yaml

```diff
@@ -237,6 +237,17 @@ CORE_SESSION_TIMEOUT:
   name: CORE_SESSION_TIMEOUT
   parent: END_JOURNEY
 
+#retrieve cri credential state
+RETRIEVE_CRI_CREDENTIAL:
```

Comment:
Can't remember if its needed exactly. But just to follow the pattern of the other states, could we also add the name and parent field in here

Reply:
Done